src/components/CertificateTemplates/tlds/sg/gov/seab/SOR_PSLE_before_1979/sor.js

```diff
@@ -0,0 +1,133 @@
+import PropTypes from "prop-types";
+import { get } from "lodash";
+import {
+  RENDER_SOR_INFO_PSLE,
+  SOR_BORDER_PSLE,
+  SOR_TOP_LOGO,
+  SOR_TOP_HEADER,
+  SOR_TITLE,
+  RENDER_SOR_HEADER,
+  RENDER_SOR_FOOTER,
+  SOR_TRANSCRIPT_FONT_SIZE_11
+} from "../common";
+
+export const renderTranscript = ({ certificate }) => {
+  // Get transcript info
+  const transcript = get(certificate, "transcript");
+
+  const transcriptDetailsPSLE = transcript.map(trn => (
+    <div className="row" style={SOR_TRANSCRIPT_FONT_SIZE_11} key={trn.id}>
```

Comment:
```suggestion
    <div className="row" style={SOR_TRANSCRIPT_FONT_SIZE_11} key={trn.name}>
```

`trn.id` does not exist and will throw an error in the front-end as the key will not be unique. Suggest to use `trn.name` if it is unique. 

Believe that similar error might occur on other templates.